### PR TITLE
feature: add temporal worker that only connects to cloud

### PR DESCRIPTION
**Pull request by fuziontech**
## Description
A temporal worker that connects to temporal cloud

## Type of change
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: charts/posthog/templates/temporal-py-worker-hpa.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ template "posthog.fullname" . }}-worker
+  labels: {{- include "_snippet-metadata-labels-common" . | nindent 4 }}
+spec:
+  scaleTargetRef:
+    kind: Deployment
+    apiVersion: apps/v1
+    name: {{ template "posthog.fullname" . }}-temporalPyWorker
```
Comment (xvello) [inline charts/posthog/templates/temporal-py-worker-hpa.yaml:11]:
```suggestion
    name: {{ template "posthog.fullname" . }}-temporal-py-worker
```

File: charts/posthog/values.yaml [hunk 2]
```diff
@@ -1087,6 +1087,53 @@ pluginsScheduler:
   # -- Sentry endpoint to send errors to. Falls back to global sentryDSN
   sentryDSN:
 
+temporalPyWorker:
+  # -- Whether to install the PostHog Temporal Python worker stack or not.
+  enabled: false
```
Comment (xvello) [inline charts/posthog/values.yaml:1092]: Let's keep disabled by default for now to be able to recover quickly if this were to break deploys.
A follow-up PR can switch to `true`.

**Review by xvello — APPROVED** [review]
Looks OK for a first phase. @tomasfarias could you please review it again?

**Review by tomasfarias — APPROVED** [review]
looks good to me, thank you!

**Pull request merged**